app/views/finders/_option_select_facet.html.erb

```diff
@@ -1,12 +1,11 @@
 <% cache_if option_select_facet.cacheable?, option_select_facet.cache_key do %>
-  <% close_facet = option_select_facet.unselected? && ((option_select_facet_counter > 0) || option_select_facet.close_facet?) %>
   <%= render partial: 'components/option-select', locals: {
     :key => option_select_facet.key,
     :title => option_select_facet.name,
     :aria_controls_id => "js-search-results-info",
     :options_container_id => option_select_facet.key,
     :options => option_select_facet.options("js-search-results-info", option_select_facet.key),
-    :closed_on_load => close_facet,
+    :closed_on_load => option_select_facet.close_facet?,
```

Comment:
I'd suggest renaming `close_facet?` to `closed_on_load?` as the former sounds like it might do something while the latter is clearer about what it does